AcloConcurrency/AcloConcurrency/Helpers/UserStorage.swift

```diff
@@ -0,0 +1,7 @@
+import SwiftUI
+public class UserStorage: ObservableObject{
```

Comment:
add space between import and UserStorage
And I think ObservableObject useless here